packages/react-jss/src/JssProvider.js

```diff
@@ -1,11 +1,23 @@
-import {Component, Children} from 'react'
+// @flow
+import {Component, Children, type Element, type ElementType} from 'react'
 import PropTypes from 'prop-types'
+import type {Jss, generateClassName as GenerateClassNameType, SheetsRegistry} from 'jss'
```

Comment:
why mapping from lower case to capital?

Reply:
ohh, I think we need to rename it to upper case in the core package in the first place

Reply:
Once it starts uppercase you don't need to add Type suffix either